crates/cranelift/src/func_environ.rs

```diff
@@ -1059,36 +936,36 @@ impl<'module_environment> FuncEnvironment<'module_environment> {
     }
 
     // Used by the typed continuations API.
-    pub fn generate_builtin_call(
-        &mut self,
-        builder: &mut FunctionBuilder,
-        index: BuiltinFunctionIndex,
-        sig: ir::SigRef,
-        args: Vec<ir::Value>,
-    ) -> (ir::Value, ir::Value) {
-        let mut args = args;
-        let (vmctx, addr) =
-            self.translate_load_builtin_function_address(&mut builder.cursor(), index);
-        args.insert(0, vmctx);
-        let call_inst = builder.ins().call_indirect(sig, addr, &args);
-        let result_value = builder.func.dfg.first_result(call_inst);
-        return (vmctx, result_value);
-    }
-
-    pub fn generate_builtin_call_no_return_val(
-        &mut self,
-        builder: &mut FunctionBuilder,
-        index: BuiltinFunctionIndex,
-        sig: ir::SigRef,
-        args: Vec<ir::Value>,
-    ) -> ir::Value {
-        let mut args = args;
-        let (vmctx, addr) =
-            self.translate_load_builtin_function_address(&mut builder.cursor(), index);
-        args.insert(0, vmctx);
-        builder.ins().call_indirect(sig, addr, &args);
-        return vmctx;
-    }
+    // pub fn generate_builtin_call(
+    //     &mut self,
+    //     builder: &mut FunctionBuilder,
+    //     index: BuiltinFunctionIndex,
+    //     sig: ir::SigRef,
+    //     args: Vec<ir::Value>,
+    // ) -> (ir::Value, ir::Value) {
+    //     let mut args = args;
+    //     let (vmctx, addr) =
+    //         self.translate_load_builtin_function_address(&mut builder.cursor(), index);
+    //     args.insert(0, vmctx);
+    //     let call_inst = builder.ins().call_indirect(sig, addr, &args);
+    //     let result_value = builder.func.dfg.first_result(call_inst);
+    //     return (vmctx, result_value);
+    // }
+
+    // pub fn generate_builtin_call_no_return_val(
+    //     &mut self,
+    //     builder: &mut FunctionBuilder,
+    //     index: BuiltinFunctionIndex,
+    //     sig: ir::SigRef,
+    //     args: Vec<ir::Value>,
+    // ) -> ir::Value {
+    //     let mut args = args;
+    //     let (vmctx, addr) =
+    //         self.translate_load_builtin_function_address(&mut builder.cursor(), index);
+    //     args.insert(0, vmctx);
+    //     builder.ins().call_indirect(sig, addr, &args);
+    //     return vmctx;
+    // }
 
```

Comment:
Let's delete this instead of commenting out

Reply:
Good catch!